benchmarks/benchmark.py

```diff
@@ -187,24 +186,17 @@ def run_single_benchmark(jmx, jmeter_args=dict(), threads=100, out_dir=None):
     else:
         # Start MMS
         docker = 'nvidia-docker' if pargs.gpus else 'docker'
-        container = 'mms_benchmark_gpu' if pargs.gpus else 'mms_benchmark_cpu'
         docker_path = 'awsdeeplearningteam/mxnet-model-server:nightly-mxnet-gpu' \
             if pargs.gpus else 'awsdeeplearningteam/mxnet-model-server:nightly-mxnet-cpu'
         if pargs.docker:
-            container = 'mms_benchmark_{}'.format(pargs.docker[0].split('/')[1])
             docker_path = pargs.docker[0]
-        run_process("{} rm -f {}".format(docker, container))
-        docker_run_call = "{} run --name {} -p 8080:8080 -p 8081:8081 -v {}:{} -itd {}".format(docker, container, MMS_BASE, DOCKER_MMS_BASE, docker_path)
+        run_process("docker rm -f mms")
+        run_process("docker pull {}".format(docker_path))
+        docker_run_call = "{} run --name mms -p 8080:8080 -p 8081:8081 -v {}:/mms -itd {}" \
```

Comment:
Since you are switching to docker commands instead of using the docker variable everywhere, maybe you should change to using `nvidia_docker = "--runtime=nvidia" if pargs.gpus else ""` and then add that to docker_run_call and hardcode the word docker. Also, doesn't docker run automatically pull if necessary?

Reply:
docker run doesn't pull the latest image if already have on in local.